friendshipper/src-tauri/src/repo/operations/download_dlls.rs

```diff
@@ -49,14 +48,16 @@ pub struct DownloadDllsOp {
     pub tx: Sender<LongtailMsg>,
     pub aws_client: AWSClient,
     pub artifact_prefix: String,
+    pub engine: Arc<T>,
```

Comment:
Is this Arc needed? The trait requires Clone/Send/Sync/`static, so I would think it would be directly accessible?

Reply:
it is, you're totally right, good catch